packages/gatsby/src/query-runner/pages-writer.js

```diff
@@ -2,6 +2,18 @@ const _ = require(`lodash`)
 const glob = require(`glob`)
 const parseFilepath = require(`parse-filepath`)
 const fs = require(`fs`)
+const path = require('path')
+const os = require('os');
+
+function joinPath(...paths) {
+  const joinedPath = path.join(...paths)
+  if(os.platform() === 'win32') {
```

Comment:
yeha but the problem is, that the string must also be escaped, because it will turn into a string literal :)